src/main/java/com/onlinebookstore/onlinebookstore/controller/BookController.java

```diff
@@ -25,28 +29,34 @@ public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "Find All Books", description = "Find All Books in DB")

```

Comment:
Maybe should say it list, for potential customer

Reply:
You mean like in description? I will think about more detailed information maybe will go with other patch